src/backend/gl/src/native.rs

```diff
@@ -253,7 +256,10 @@ impl SwapchainImage {
                 num_levels: 1,
                 num_layers: 1,
             },
-            view: ImageView::Renderbuffer(renderbuffer),
+            view: ImageView::Renderbuffer {
+                raw: renderbuffer,
+                aspects: format::Aspects::empty(),
```

Comment:
any reason to not use `::COLOR` here?

Reply:
I'm matching the aspect above in the format :)

Reply:
oh my!